jena-fuseki2/jena-fuseki-core/src/main/java/org/apache/jena/fuseki/servlets/ActionPrefixesRW.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.jena.fuseki.servlets;
+
+import org.apache.jena.atlas.logging.FmtLog;
+import org.apache.jena.riot.WebContent;
+import org.apache.jena.riot.web.HttpNames;
+
+import org.apache.jena.fuseki.servlets.prefixes.PrefixUtils;
+
+import java.io.IOException;
+
+public class ActionPrefixesRW extends ActionPrefixesR {
+
+    public ActionPrefixesRW() {}
+
+    @Override
+    protected void doOptions(HttpAction action) {
+        ActionLib.setCommonHeadersForOptions(action);
+        action.setResponseHeader(HttpNames.hAllow, "GET,OPTIONS,POST");
+        ServletOps.success(action);
+    }
+
+    public void validateDelete(HttpAction action) {
+        validate(action);
+        String prefixToRemove = action.getRequestParameter("removeprefix");
+        if (prefixToRemove == null || prefixToRemove.isEmpty() || !PrefixUtils.prefixIsValid(prefixToRemove)) {
+            ServletOps.errorBadRequest("Remove operation unsuccessful!");
+            return;
+        }
+    }
+
+    @Override
+    protected void doDelete(HttpAction action) {
+        validateDelete(action);
+        action.beginWrite();
+
+        try {
+            String prefixToRemove = action.getRequestParameter("removeprefix");
```

Comment:
Since this is a `DELETE` request would it make more sense to just use the same `prefix` parameter as the read endpoints?

The use of HTTP verb already differentiates this request from a read request so using a different parameter name seems unnecessary